### PR TITLE
Update smb-over-quic.md

### PR DESCRIPTION
**Pull request by DaPadrePedro**
1) HOST files: must be HOSTS files
2) "Click here to configure settings": the punctuation was not mentioned => "Click here to configure settings."
3) ECDCA_P256: must be "ECDSA_P256" AFAIK
4) "File Shares (SMB server)": no capital for "Shares" => "File shares (SMB server)"
5) "Windows Server 2022 Datacenter: Azure Edition": I thought the official name was "Windows Server 2022 Datacenter Azure Edition" (so without the colon)?

### CODE REVIEW

File: WindowsServerDocs/storage/file-server/smb-over-quic.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ ms.localizationpriority: medium
 
 # SMB over QUIC (PREVIEW)
 
->Applies to: Windows Server 2022 Datacenter: Azure Edition Preview, Windows 11 Insider Preview 
+>Applies to: Windows Server 2022 Datacenter Azure Edition Preview, Windows 11 Insider Preview
```
Comment (nedpyle) [inline WindowsServerDocs/storage/file-server/smb-over-quic.md:14]: Please revert, this is not correct. The name of the OS contains a colon, the name returned INSIDE the OS via API will contain no colon due to it causing application incompatibilities.

File: WindowsServerDocs/storage/file-server/smb-over-quic.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ A file server administrator must opt in to enabling SMB over QUIC. It isn't on b
 
 To use SMB over QUIC, you need the following things:
 
-- A file server running Windows Server 2022 Datacenter: Azure Edition Preview ([Microsoft Server Operating Systems Preview](https://aka.ms/ws2022ae))
+- A file server running Windows Server 2022 Datacenter Azure Edition Preview ([Microsoft Server Operating Systems Preview](https://aka.ms/ws2022ae))
```
Comment (nedpyle) [inline WindowsServerDocs/storage/file-server/smb-over-quic.md:35]: Revert

File: WindowsServerDocs/storage/file-server/smb-over-quic.md [hunk 3]
```diff
@@ -47,7 +47,7 @@ To use SMB over QUIC, you need the following things:
     - Purpose: Server Authentication (EKU 1.3.6.1.5.5.7.3.1)
     - Signature algorithm: SHA256RSA *(or greater)*
     - Signature hash: SHA256 *(or greater)*
-    - Public key algorithm: ECDCA_P256 *(or greater. Can also use RSA with at least 2048 length)*
+    - Public key algorithm: ECDSA_P256 *(or greater. Can also use RSA with at least 2048 length)*
```
Comment (nedpyle) [inline WindowsServerDocs/storage/file-server/smb-over-quic.md:50]: This is correct, I had a typo. Good catch

File: WindowsServerDocs/storage/file-server/smb-over-quic.md [hunk 4]
```diff
@@ -83,10 +83,10 @@ To use SMB over QUIC, you need the following things:
 
 ### Step 2: Configure SMB over QUIC
 
-1. Deploy a [Windows Server 2022 Datacenter: Azure Edition](https://aka.ms/ws2022ae) preview server.
+1. Deploy a [Windows Server 2022 Datacenter Azure Edition](https://aka.ms/ws2022ae) preview server.
```
Comment (nedpyle) [inline WindowsServerDocs/storage/file-server/smb-over-quic.md:86]: revert

File: WindowsServerDocs/storage/file-server/smb-over-quic.md [hunk 5]
```diff
@@ -83,10 +83,10 @@ To use SMB over QUIC, you need the following things:
...
-1. Join your Windows Server 2022 Datacenter: Azure Edition file server to your Active Directory domain and make it accessible to Windows Insider clients on the Azure public interface by adding a firewall allow rule for UDP/443 inbound. Do **not** allow TCP/445 inbound to the file server. The file server must have access to at least one domain controller for authentication, but no domain controller requires any internet access.
-1. Connect to the server with Windows Admin Center and click the **Settings** icon in the lower left. In the **File Shares (SMB server)** section, under **File sharing across the internet with SMB over QUIC**, click **Configure**.
+1. Join your Windows Server 2022 Datacenter Azure Edition file server to your Active Directory domain and make it accessible to Windows Insider clients on the Azure public interface by adding a firewall allow rule for UDP/443 inbound. Do **not** allow TCP/445 inbound to the file server. The file server must have access to at least one domain controller for authentication, but no domain controller requires any internet access.
```
Comment (nedpyle) [inline WindowsServerDocs/storage/file-server/smb-over-quic.md:88]: revert

File: WindowsServerDocs/storage/file-server/smb-over-quic.md [hunk 6]
```diff
@@ -83,10 +83,10 @@ To use SMB over QUIC, you need the following things:
...
+1. Connect to the server with Windows Admin Center and click the **Settings** icon in the lower left. In the **File shares (SMB server)** section, under **File sharing across the internet with SMB over QUIC**, click **Configure**.
```
Comment (nedpyle) [inline WindowsServerDocs/storage/file-server/smb-over-quic.md:89]: this is correct, good eye

File: WindowsServerDocs/storage/file-server/smb-over-quic.md [hunk 7]
```diff
@@ -103,7 +103,7 @@ For a demonstration of configuring and using SMB over QUIC, watch this video:
 
 ### Step 3: Connect to SMB shares
 
-1. Join your Windows 11 Insider Preview to your domain.Be certain the names of the SMB over QUIC file server's certificate subject alternative names are published to DNS and are fully qualified **OR** added to the HOST files for your Windows 11 Insider Preview. Ensure that the server's certificate subject alternative names are published to DNS **OR** added to the HOSTS files for your Windows 11 Insider Preview.
+1. Join your Windows 11 Insider Preview to your domain.Be certain the names of the SMB over QUIC file server's certificate subject alternative names are published to DNS and are fully qualified **OR** added to the HOSTS files for your Windows 11 Insider Preview. Ensure that the server's certificate subject alternative names are published to DNS **OR** added to the HOSTS files for your Windows 11 Insider Preview.
```
Comment (nedpyle) [inline WindowsServerDocs/storage/file-server/smb-over-quic.md:106]: Typo here, need a space between domain and be. The HOSTS change is correct, good catch

**Comment by IngridAtMicrosoft** [comment]
#assign:IngridAtMicrosoft

**Comment by IngridAtMicrosoft** [comment]
@DaPadrePedro Thank you for finding those typos!  In the future, please use a GitHub issue. You can still edit pages that way, but it will also give us some metrics on the type and frequency of Docs problems. Thanks again!

File: WindowsServerDocs/storage/file-server/smb-over-quic.md (hunk 1 above)
Comment (DaPadrePedro) [reply WindowsServerDocs/storage/file-server/smb-over-quic.md:14]: Oops, you're right! My mistake!

**Review by nedpyle — APPROVED** [review]
Looks awesome. Thanks for catching all these mistakes, I really appreciate the contribution!

**Comment by DaPadrePedro** [comment]
You're welcome, Ned! I just want to contribute wherever I can, towards the highest quality and with no pedantic intention at all. Thank you for reviewing my proposals!

**Comment by IngridAtMicrosoft** [comment]
@DaPadrePedro Yes that is one way to do it, and you can certainly keep doing it that way.  However, we get additional metrics when a GitHub issue is reported.  Continue doing what you are comfortable with.

**Comment by PRMerger5** [comment]
@IngridAtMicrosoft: I'm sorry - only the author of this article, @NedPyle, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by IngridAtMicrosoft** [comment]
@nedpyle Please sign off on this change.

**Pull request merged**